docs/development.rst

```diff
@@ -11,7 +11,7 @@ I am the primary author of Hypothesis.
 2. Fixing bugs and other code health issues
 3. Improving documentation
 4. General release management work
-5. Planning the general roadmap of the project
+5. Planning the general road map of the project
```

Comment:
Well, this case I checked and it seems to be in two words: http://www.oxforddictionaries.com/definition/english/road-map?q=roadmap

However, it's an edge case for me since roadmap is also well known word that defines quite special things.